tests/php/IntegrationTest.php

```diff
@@ -60,7 +60,7 @@ protected function doIntegrationTest(string $filename, string $phelCode, string
             );
         }
         $compiledCode = trim(implode('', $compiledCode));
-        $this->assertEquals($generatedCode, $compiledCode, 'in ' . $filename);
+        self::assertEquals($generatedCode, $compiledCode, 'in ' . $filename);
```

Comment:
Why is this static now?

Reply:
Because all asssertion (in PHPUnit at least) are static functions:
```php
public static function assertEquals($expected, $actual, string $message = ''): void
```
I thought it might be interesting to agree on one style of writing them. For this reason, I wrote Sebastian asking for this particular topic, what does he recommend: https://twitter.com/Chemaclass/status/1284800657027936256

From my point of view, if a function is static, I would prefer to call it using either `self::` of `static::`, this way it will visually clear that this function is static from the calling context.

Reply:
We got the answer from Sebastian: https://twitter.com/s_bergmann/status/1284810322675630080
You @jenshaase have the last word here :)

Reply:
Use `$this`